history-of-the-quakers-v3/original/25-book-6-chapter-5.adoc

```diff
@@ -413,15 +417,25 @@ as you are not and cannot be forgotten in my addresses and approaches to him,
 who rest in his unchangeable love, Dear friends and brethren,
 your`'s unalterably in the communion of the precious truth,
 
-2nd 1st mo 1689
-
+[.signed-section-signature]
 William Penn
 
+[.signed-section-context-close]
+2nd 1st mo 1689
+
+[.signed-section-context-close]
```

Comment:
I sort of lean towards making this a postscript, although it doesn't fit perfectly in either category.